WcaOnRails/app/views/competitions_mailer/notify_organizer_of_addition_to_competition.html.erb

```diff
@@ -0,0 +1,15 @@
+<p>
+  <%= t 'users.mailer.organizer_addition_email.intro', name: @organizer.name %>
+</p>
+
+<p>
+  <%= t 'users.mailer.organizer_addition_email.information', confirmer: @confirmer.name, competition: @competition.name %>
+  <%= link_to 'Competition edit page', edit_competition_url(@competition) %>
+</p>
+
+<p>
+  <%= t 'users.mailer.competition_submission_email.contact' %>
+  <% for email in @competition.delegates.pluck(:email) %>
```

Comment:
`for ... in` leaks the "local" variable, so it's better not to use it:

```
irb(main):001:0> for x in [1,2]
irb(main):003:1> end
=> [1, 2]
irb(main):004:0> x
=> 2
```

It also used to be (around Ruby 2.0) slower than using `#each`, but this might be fixed in newer Ruby versions.